### ISSUE TITLE
The website's default accent color is not WCAG AA compliant

### ISSUE THREAD

**Issue by angeloanan** [discordextremelist/website#84]
**Describe the Bug**
The default website accent color (RGB 177, 20, 255 or HEX #b114ff) is not WCAG compliant. This may result in a rough navigation experience for the color blind and people with small monitors / monitors with bad color.

Consider bumping the contrast ratio to atleast 4.5:1. [More info](https://web.dev/color-contrast/)

**Screenshots**
Taken on Windows 10, 1080p screen with ClearType turned on
![image](https://user-images.githubusercontent.com/2129163/88602409-f4dfba80-d09c-11ea-82cd-78d1d23f51ce.png)

**Comment by angeloanan** [comment]
Also the same with the danger text (CSS Class `.is-danger`)

![image](https://user-images.githubusercontent.com/2129163/88603124-6835fc00-d09e-11ea-8f4b-28759a904862.png)

**Comment by advaith1** [comment]
related: #80

**Comment by advaith1** [comment]
looks like our current default has good contrast in our titles (incl card titles) due to the size, but not in inline code, where we also use the accent color.

**Comment by advaith1** [comment]
possible replacement: #ba2eff

related: https://crbug.com/1120223

**Comment by Cairo2k18** [comment]
> possible replacement: #ba2eff

looks bad when used as bg colour, foreground text has to be black to meet WCAG compliance

**Comment by Cairo2k18** [comment]
Solved in v5.4.20
